Dockerfiles/ccp-api-server/Dockerfile

```diff
@@ -17,10 +17,14 @@ ENV PYTHONPATH="/opt/ccp_api_server"
 
 COPY . /opt/ccp_api_server
```

Comment:
I am thinking, now that `/opt/ccp_api_server` hosts the complete service code, we should name it as our service repo name or `ccp-openshift` as we did in slave container.

Reply:
It did right from the start